lib/constants/constants.dart

```diff
@@ -3,14 +3,36 @@ import 'dart:core';
 
 class DesignColors {
   static const Color naviColor = Color.fromARGB(255, 0, 131, 238);
-  static const Color kBackgroundColor = Colors.white;
+  static const Color kBackgroundColor = Color.fromARGB(255, 248, 248, 248);
 }
 
 class AppStyle {
   static const double cornerRadius = 20;
   static const double kDefaultPadding = 20.0;
 }
 
+class AppInputBorders {
+  static const OutlineInputBorder border = OutlineInputBorder(
```

Comment:
wenn die Klasse schon Borders im Namen hat, kannst du "border" in den Membern eigentlich weglassen.
Also statt border => default
statt focusedBorder => focused
oder?